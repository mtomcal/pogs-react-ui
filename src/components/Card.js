import React, {Component} from "react";
// import PropTypes from 'prop-types';

class Card extends Component {
  constructor(props) {
    super(props);
    this.state = {};
  }

  render() {
    return (
      <div className="panel panel-default">
        <div className="panel-body">
          <div className="col-sm-12">
            <div className="col-sm-12">
              {this.props.children}
            </div>
          </div>
        </div>
      </div>
    );
  }
}

Card.propTypes = {};

export default Card;
