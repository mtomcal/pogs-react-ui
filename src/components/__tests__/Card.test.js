import React from 'react';
import Card from '../Card';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

it('renders with two elements', () => {
  const component = renderer.create(
    <Card>
      <h1>Stuff</h1>
      <p>More stuff</p>
    </Card>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
