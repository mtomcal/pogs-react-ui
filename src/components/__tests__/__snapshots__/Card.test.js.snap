exports[`test renders with two elements 1`] = `
<div
  className="panel panel-default">
  <div
    className="panel-body">
    <div
      className="col-sm-12">
      <div
        className="col-sm-12">
        <h1>
          Stuff
        </h1>
        <p>
          More stuff
        </p>
      </div>
    </div>
  </div>
</div>
`;
